package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jupyterlab-topbar-extensions",
-  "version": "0.6.1",
+  "version": "1.0.0",
```

Comment:
I guess we'll still need to update this version?

Looking at the logs of the run for this PR, the title for the release is still `1.0.0` in the check release run:

 https://github.com/jupyterlab-contrib/jupyterlab-topbar/actions/runs/6313993662/job/17143290814?pr=101

![image](https://github.com/jupyterlab-contrib/jupyterlab-topbar/assets/591645/a66b9fe8-778f-4957-a094-e4d4b5961598)

Reply:
As we removed the top level python project, `bump version` step *will not* bump the version of top level `package.json` which is eventually used in release titles. So, we are doing it manually to the version that we will use in the `jupyter-releaser` which is `1.0.0`.  

For the next minor version, we will need to manually bump the top level `package.json` to `1.1.0` so that the release titles will contain `1.1.0`. Else, release titles will *always* have current `0.6.1`. Does it make sense?

Reply:
Yeah but ideally this version bump should be handled by the releaser.

Otherwise if we need to update this version manually every time then we might as well version all packages manually in a dedicated PR as mentioned in https://github.com/jupyterlab-contrib/jupyterlab-topbar/issues/98#issuecomment-1735552955?

Reply:
> Yeah but ideally this version bump should be handled by the releaser.

Agree! We can have a hook to do it based on env var `RH_VERSION_SPEC`.

> Otherwise if we need to update this version manually every time then we might as well version all packages manually in a dedicated PR as mentioned in https://github.com/jupyterlab-contrib/jupyterlab-topbar/issues/98#issuecomment-1735552955?

As I said in the [comment](https://github.com/jupyterlab-contrib/jupyterlab-topbar/issues/98#issuecomment-1735601301), the *unbumped* packages will always be overwritten in the registry, right? If you think that is not going to be an issue, we can bump versions manually in a dedicated PR.

Reply:
Sorry @jtpio, I was bit occupied last few days. So, how do we do this? Bump versions manually and skip `bump-version` step in releaser workflows? 

If so, how do we manage the version of the top level package? Make a bump everytime we bump version of _atleast_ one of the packages just to create a unique title in releases?